frontend/src/concepts/pipelines/apiHooks/mlmd/__tests__/useArtifactsFromMlmdContext.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { testHook, standardUseFetchState } from '~/__tests__/unit/testUtils/hooks';
+import { Context, MetadataStoreServicePromiseClient } from '~/third_party/mlmd';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { useArtifactsFromMlmdContext } from '~/concepts/pipelines/apiHooks/mlmd/useArtifactsFromMlmdContext';
+
+// Mock the usePipelinesAPI hook
+jest.mock('~/concepts/pipelines/context', () => ({
+  usePipelinesAPI: jest.fn(),
+}));
+
+// Mock the MetadataStoreServicePromiseClient
+jest.mock('~/third_party/mlmd', () => {
+  const originalModule = jest.requireActual('~/third_party/mlmd');
+  return {
+    ...originalModule,
+    MetadataStoreServicePromiseClient: jest.fn().mockImplementation(() => ({
+      getArtifactsByContext: jest.fn(),
+    })),
+  };
+});
+
+describe('useArtifactsFromMlmdContext', () => {
+  const mockClient = new MetadataStoreServicePromiseClient('');
+  const mockUsePipelinesAPI = usePipelinesAPI as jest.Mock;
+  const mockGetArtifactsByContext = mockClient.getArtifactsByContext as jest.Mock;
```

Comment:
Just FYI, you should use `jest.mocked` where possible to ensure that we get type support instead of `jest.Mock`.

Although if you're working with partial data then `jest.mocked` can make things more complicated.